aws-java-sdk-dynamodb/src/test/java/com/amazonaws/services/dynamodbv2/datamodeling/DynamoDBMapperTest.java

```diff
@@ -0,0 +1,29 @@
+package com.amazonaws.services.dynamodbv2.datamodeling;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+import com.amazonaws.services.dynamodbv2.model.DeleteTableRequest;
+
+public class DynamoDBMapperTest {
+
+    @Test
+    public void testGenerateDeleteTableRequest(){
```

Comment:
Kudos writing a test! This is testing the override path, could you write a similar test that does not use the override and relies on the table name in the annotation to cover both scenarios.